src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java

```diff
@@ -11,8 +11,10 @@
 import java.util.Optional;
 import java.util.function.Predicate;
 
+import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.ONE_V_ONE;
```

Comment:
One_V_One in MultipartyScenario enum looks odd

Reply:
This existed prior to my changes and is used quite a lot throughout civil-service
Not sure of how much work it would take to refactor this but seems to be out of scope for this ticket